Gems/Atom/RPI/Code/Source/RPI.Reflect/Asset/AssetUtils.cpp

```diff
@@ -89,21 +89,26 @@ namespace AZ
             AsyncAssetLoader::~AsyncAssetLoader()
             {
                 Data::AssetBus::Handler::BusDisconnect();
+                SystemTickBus::Handler::BusConnect();
```

Comment:
The system tick bus is not protected by a mutex (its event queue is though).
Are we sure this cannot happen in a non-main thread?  (that is, connecting, disconnecting)

Reply:
Actually there's a typo in that line of the `AsyncAssetLoader` destructor. it should be calling `SystemTickBus::Handler::BusDisconnect()`.

I don't anticipate a deadlock here because creation and destruction of AsyncAssetLoader is always on main thread, and so far no lambdas enqueued on the `Secondary Copy Queue` thread, used by `StreamingImage->AsyncUploadQueue`, don't connect/disconnect to/from SystemTickBus.